### PR TITLE
[16.0][MIG] account_invoice_warn_message: Migration to 16.0

### PR DESCRIPTION
**Pull request by RogerSans**
Standard Migration
T-6530

@HaraldPanten 
@ValentinVinagre 
@Jaimermaccione

### CODE REVIEW

**Review by Jaimermaccione — APPROVED** [review]
@RogerSans LGMT!. The module has been tested on Runboat and works well. 

There's just a small visual detail that could be improved if it's not too costly to modify. The information icon is placed too close to the configured text in the client. I’ve compared versions, and in v17, it’s also close, but in v15, there's a space between the icon and the text, which gives a better visual appearance. Functionally, it’s perfect, so I’m giving it my approval, but that change would leave a better finish.

Test on V15:
image
Test on v17:
image

**Comment by ValentinVinagre** [comment]
> @RogerSans LGMT!. The module has been tested on Runboat and works well.
> 
> There's just a small visual detail that could be improved if it's not too costly to modify. The information icon is placed too close to the configured text in the client. I’ve compared versions, and in v17, it’s also close, but in v15, there's a space between the icon and the text, which gives a better visual appearance. Functionally, it’s perfect, so I’m giving it my approval, but that change would leave a better finish.
> 
> Test on V15: image Test on v17: image

@RogerSans 
Could you, if it is not too expensive, make the change for both versions? I agree that it improves the display.

**Comment by RogerSans** [comment]
Can you review the visual solution pls?
@ValentinVinagre 
@Jaimermaccione

**Review by Jaimermaccione — APPROVED** [review]
@RogerSans LGTM! Functional review on runboat!

**Comment by ValentinVinagre** [comment]
@RogerSans  Try to have the icon above the text and not at the bottom and make sure the full width is used in the display, since currently only half of the space is used.
image

**Comment by RogerSans** [comment]
Changed to full use in the display warning.
Selección_010

@ValentinVinagre

File: account_invoice_warn_message/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- Héctor Villarreal \<<hector.villarreal@forgeflow.com>\>
+- Manuel Regidor \<<manuel.regidor@sygel.es>\>
```
Comment (ValentinVinagre) [inline account_invoice_warn_message/readme/CONTRIBUTORS.md]: Use this format please:

https://github.com/OCA/l10n-spain/blob/6c9a473b1607f52da7295fafaaae4eae70149f32/l10n_es_aeat_mod303/readme/CONTRIBUTORS.md?plain=1#L9-L11

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Some comments:

1. You're mixing pre-commit stuff and Migration changes in the same commit. You should separate them.
2. The tag in the migration commit [FWD] is not correct. You have to change it by [MIG]. The same for the PR title.

THX!

**Comment by RogerSans** [comment]
@HaraldPanten 
Solved 
THX

**Review by HaraldPanten — APPROVED** [review]
LGTM.

My fault when I said my comments about the pre-commit, as it is a migration from V17 to V16 and there are no pre-commit changes.

**Pull request merged**